.gitignore

```diff
@@ -6,4 +6,4 @@ node_modules/
 .vscode/
 docs/.vuepress/dist/
 package-lock.json
-
+yarn.lock
```

Comment:
Don't ignore yarn lockfile please. Since alpha version program may come up with many unexpected issues, you should keep lockfile in you repository to make sure that ci will use exactly the same version.